### PR TITLE
[Documentation]: Updated Docs for Bank - Bank Transactions #952

### PR DESCRIPTION
**Pull request by UweVanOpfern**
Issue #952

My account number

593c3874ca9b04dc1386b0ef7f72e075d3fd729fb262f816d7bde528e9047168

### CODE REVIEW

**Comment by shahraizali** [comment]
#952

**Comment by UweVanOpfern** [comment]
This issue, Update Docs for Primary Validator - Confirmation Blocks #966 fixed in this PR as well

File: src/containers/BankApi/BankApiBankTransactions/index.tsx [hunk 1]
```diff
@@ -14,48 +14,54 @@ const BankApiBankTransactions: FC = () => {
...
-    "block": {
-      "id": "e00c5522-1b73-4a46-bd03-629d446eec19",
-      "created_date": "2020-07-14T03:14:36.436771Z",
-      "modified_date": "2020-07-14T03:14:36.436796Z",
-      "balance_key": "efa253d24ee516fe5ed45bb4e47a3146026e97f766df1cdb13663ec62174e214",
-      "sender": "0cdd4ba04456ca169baca3d66eace869520c62fe84421329086e03d91a68acdb",
-      "signature": "a1bbd321ad6d3f74f027de5a2c19457779fe1466708c2ea3dc9bf993a4048471025928cabcea934cf681930657ff661f05452f34630e5ea45d962e828899af00"
+        code={`{
+  "count": 2116,
+  "next": "http://143.110.137.54/bank_transactions?limit=3&offset=53",
```
Comment (shahraizali) [inline src/containers/BankApi/BankApiBankTransactions/index.tsx:34]: this url 
```http://143.110.137.54/bank_transactions?limit=3&offset=53```
should have **limit** set to 2 and set the offset to an even number as well if you like.

Comment (UweVanOpfern) [reply src/containers/BankApi/BankApiBankTransactions/index.tsx]: I fixed it

**Review by shahraizali — APPROVED** [review]

**Comment by Hristijan95** [comment]
@UweVanOpfern , the JSON should be formatted with 2 spaces indents as specified in the issue, please format it when you get the chance.

**Comment by UweVanOpfern** [comment]
@Hristijan95, am gonna fix them, thank you

**Comment by DucPhamTV** [comment]
POST requests to PV /confirmation_blocks is supported. So the document should update it as well.

**Comment by Rajputusman** [comment]
https://github.com/thenewboston-developers/Website/issues/966#issuecomment-735366599

**Pull request merged**